source/hic_hal/nxp/lpc11u35/uart.c

```diff
@@ -99,6 +99,9 @@ int32_t uart_set_configuration(UART_Configuration *config)
     baudrate = config->Baudrate;
     // Compute baud rate dividers
     mv = 15;
+#ifdef BOARD_RTL8195AM
+    config->Baudrate = 38400;
```

Comment:
If you can, please determine if this is a bug or not. This either needs to be fixed or there needs to be an explanation of why it can't be fixed and needs this workaround.